src/yadism/coefficient_functions/light/kernels.py

```diff
@@ -77,7 +95,7 @@ def generate(esf, nf, skip_heavylight=False):
     return [ns, g, s]
 
 
-def nc_weights(coupling_constants, Q2, kind, nf, skip_heavylight=False):
+def nc_weights(coupling_constants, Q2, kind, nf, is_pv, skip_heavylight=False):
```

Comment:
In principle, being parity preserving or parity violating is a property of the `kind`, so you're passing twice this information, and you may avoid the extra argument.

In order to make it simpler, consider to lift the `is_parity_violating()` function from an `ObservableName` property to a standalone function on kinds (i.e. on `str`s), and use this latter function in the current property implementation.
Consequently, you could use the function on kinds even in here, without duplication.

Reply:
Actually I now notice that `kind` is not used anymore in this function. So we can actually drop it and it's even simpler.

Reply:
solved in 28b50af3222fa7560d648053cfb5fc081adfdf2b

Reply:
So much the better :)